phoenix-core/src/it/java/org/apache/phoenix/tx/TxCheckpointIT.java

```diff
@@ -104,9 +105,9 @@ public void testUpsertSelectDoesntSeeUpsertedData() throws Exception {
         conn.createStatement().execute("CREATE "+(localIndex? "LOCAL " : "")+"INDEX " + indexName + " ON " + fullTableName + "(val)");
```

Comment:
while you are at it, Could you please logically separate this line of code? Lots of things happening in one line.

Thanks.